### PR TITLE
Cypress test

### PR DESCRIPTION
**Pull request by huyenuet**
## Features

- [x] Setup cypress
- [x] Add cypress e2e test

### CODE REVIEW

**Comment by huyenuet** [comment]
Sure @nvh95

**Comment by nvh95** [comment]
Can you revert these changes in `package-lock.json` as well?
@ntt261298 Can you guide @huyenuet on this? Thanks.

**Comment by nvh95** [comment]
@huyenuet Please help to commit the change in `website/package-lock.json` as well. Thanks.

**Comment by nvh95** [comment]
After a while of consideration, I think we shouldn't put `cypress` code in `website` folder. The reason is the deployment on Netlify will take longer since it needs to install `cypress` (which is a big package). I don't see that Netlify offer us to opt-out devDependencies packages in Netlify Dashboard.

Moreover, since the new cypress code interact with the actual website on www.jest-preview.com. It's not a part of website source code. We can create a new folder under root and name it something like `e2e-docs`.

**Comment by huyenuet** [comment]
@nvh95 I updated the PR per your request. The new folder name was set to "docs-e2e-testing".

File: docs-e2e-testing/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "e2e-testing-for-docs",
+  "version": "1.0.0",
```
Comment (nvh95) [inline docs-e2e-testing/package.json:3]: Can you change to 
```suggestion
  "version": "0.0.1",
```

File: docs-e2e-testing/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "e2e testing for JestPreview Docs website",
+  "main": "index.js",
+  "scripts": {
+    "cypress-test": "cypress open"
+  },
+  "author": "huyenuet",
+  "license": "ISC",
```
Comment (nvh95) [inline docs-e2e-testing/package.json:10]: Can you change to `MIT`

**Review by nvh95 — COMMENTED** [review]
Your PR looks great so far. I left 2 small comments.

**Comment by nvh95** [comment]
Thank you very much @huyenuet. Welcome to contributors 🎉🎉🎉
@all-contributors please add @huyenuet for test

**Comment by nvh95** [comment]
@nvh95 
- [ ] To add GitHub Actions to run it periodically.

**Pull request merged**